autoload/nvimgdb.vim

```diff
@@ -2,15 +2,16 @@
 function! nvimgdb#TermOpen(command, tab)
   return termopen(a:command,
     \ {'on_stdout': {j,d,e -> luaeval("NvimGdb.parser_feed(_A[1], _A[2])", [a:tab, d])},
-    \  'on_exit': {j,c,e -> execute('if c == 0 | silent! close! | endif')},
+    \  'on_exit': {j,c,e -> execute(printf("if c == 0 | sil! bw! | endif" .
```

Comment:
Pretty much action already. Would you be able to put it all into a function either in vimscript `nvimgdb#` or in Lua `NvimGdb`?